contracts/BatchNFT.sol

```diff
@@ -27,6 +27,16 @@ contract BatchNFT is ERC721, Ownable {
         setSupply(supply_);
     }
 
+    receive() external payable {}
```

Comment:
What is the point of this function?

Reply:
just make the contract can receive Ether, now only for [testing](https://github.com/thematters/matters-avatar/pull/34/files#diff-ee4c354a2a46ddb89a4678af8d6e4749fefefd658910e922fe70c2ff2f965b64R75)